### ISSUE TITLE
"invalid gzip header" on first execution

### ISSUE THREAD

**Issue by lobisquit** [tectonic-typesetting/tectonic#135]
Hi everybody!

The title is self explanatory: on the first run on tectonic in a `.tex` file this error pops up.

```
note: this is a BETA release; report issues at https://github.com/tectonic-typesetting/tectonic/issues
note: indexing https://dl.bintray.com/pkgw/tectonic/tl2016extras/2016.0r4/tlextras-2016.0r4.tar
note: resolved to https://akamai.bintray.com/59/59eecf9d49e259191ef66cb68b274c7c50fa20a75f593130d2603176418422eb?__gda__=exp=1510852326~hmac=449ba22d7fbe8f9451ead318155bff047495650de7ca08930914c905a04138aa&response-content-disposition=attachment%3Bfilename%3D%22tlextras-2016.0r4.tar%22&response-content-type=application%2Fx-tar&requestInfo=U2FsdGVkX187Qgm3s2_N9A8_q193GPXkXfOMgvnFUmXtVhtSTn8ekq9vMxJoui0QP8xTlLOyA-5bvhYaK4zwPGQoH4x95TI21WUBXKRvjyUcXOr7FHr6UuY1DT53h7B1eEOICmB8XFnFARUPg2ZmXQ&response-X-Checksum-Sha1=2955af2fd50eb7ce6af55fd926943b79e525fde0&response-X-Checksum-Sha2=59eecf9d49e259191ef66cb68b274c7c50fa20a75f593130d2603176418422eb
error: invalid gzip header
```

I have seen this error was given to some people in the past, so I read #113, #21 and #59 and tried the proposed solutions, for example adding the following lines to `config.toml` file, but with no results.

```
[[default_bundles]]
url = "https://dl.bintray.com/pkgw/tectonic/tl2016extras/2016.0r4/tlextras-2016.0r4.tar"
```

I tried to download to the "resolved url" proposed but tectonic and the `tar` file seems to download just fine, so I have no clue how to troubleshoot this.

I tried also to switch to a different network (my phone) and error is the same.

Any idea how to debug this? Thank you for your time :wink:

**Comment by lobisquit** [comment]
I forgot to mention, but I am using Arch Linux and I installed tectonic via AUR.

**Comment by lobisquit** [comment]
Ok, I have found the issue: the AUR package is not up-to-date.
Fixing the pkgver to the current one (1.6.0) fixes the issue.

**Comment by pkgw** [comment]
That is a strange failure mode to me! But if that fixes it, OK.

I actually just released version 0.1.7. We set up infrastructure to automatically make new AUR packages when new versions come out, but I am not an Arch user so I am not sure if it is working.

**Comment by lobisquit** [comment]
It is a nice idea to setup automatic packaging :+1: 
However it seems not to sync properly, as you can see [here](https://aur.archlinux.org/packages/tectonic) in the AUR.
